modules/account-lib/src/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,55 @@
+import { bigIntFromBufferBE, bigIntToBufferBE } from '../util';
```

Comment:
I wonder if we should make a `bigIntFromU8ABE` utility function instead of having to do `Buffer.from()` all over the place.

Reply:
Good Suggestion, Updated